metrics.go

```diff
@@ -82,6 +82,15 @@ type Metrics struct {
 
 	// Number of times baggage restrictions failed to update.
 	BaggageRestrictionsUpdateFailure metrics.Counter `metric:"baggage_restrictions_updates" tags:"result=err"`
+
+	// Number of times debug spans were throttled.
+	ThrottledDebugSpans metrics.Counter `metric:"throttled_debug_spans"`
+
+	// Number of times throttler successfully updated.
+	ThrottlerUpdateSuccess metrics.Counter `metric:"throttler_update_success" tags:"result=ok"`
```

Comment:
the tag `result=ok` kinda defeats naming the metric throttler_update_success

It should be `metric:"throttler_updates" tags:"result=ok"` and `metric:"throttler_updates" tags:"result=err"`

Reply:
Ya I had no idea about these tags. Thanks for the explanation.